.docs/content/2.usage/use-armonik-api-in-an-angular-app.md

```diff
@@ -0,0 +1,549 @@
+
+```bash
+pnpm install @ngx-grpc/common @ngx-grpc/core @ngx-grpc/grpc-web-client google-protobuf @types/google-protobuf @aneoconsultingfr/armonik.api.angular
+```
+
+`google-protobuf` is a dependency that will be used to serialize and deserialize the messages.
+
+::alert{type="info"}
+You can read more about `google-protobuf` in the [official documentation](https://www.npmjs.com/package/google-protobuf).
+::
+
+`@ngx-grpc/common` `@ngx-grpc/core` `@ngx-grpc/grpc-web-client` are dependencies that will allow us to use gRPC in our Angular App.
+
+::alert{type="info"}
+You can read more about `@ngx-grpc` in the [official documentation](https://github.com/ngx-grpc/ngx-grpc).
+::
+
+`@aneoconsultingfr/armonik.api.angular` is the package that contains the ArmoniK API (definition of the gRPC services and the generated code).
+
+::alert{type="info"}
+You can read more about `@aneoconsultingfr/armonik.api.angular` in the [package documentation](https://www.npmjs.com/package/@aneoconsultingfr/armonik.api.angular).
+::
+
+## Configure the Angular App
+
+In order to be able to call some gRPC services, we need to configure the Angular App to use gRPC. In the config file, we will import the providers from the `@ngx-grpc/core` and `@ngx-grpc/grpc-web-client` packages.
+
+In the `src/app/app.config.ts` file, you can add the following content:
+
+```typescript [app.config.ts]
+import { ApplicationConfig, importProvidersFrom } from '@angular/core';
+import { GrpcCoreModule } from '@ngx-grpc/core';
+import { GrpcWebClientModule } from '@ngx-grpc/grpc-web-client';
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    importProvidersFrom(GrpcCoreModule.forRoot()),
+    importProvidersFrom(GrpcWebClientModule.forRoot({ settings: { host: '' } }))
+  ]
+};
+```
+
+That's it! Now, we are ready to do some gRPC calls!
+
+## Call the gRPC services
+
+In this section, we will get partitions  ArmoniK.
+
+::alert{type="info"}
+We could have used any other service. We chose to use the `ListPartitions` service because it is a simple service and we are sure that it will return data. If you want to use another service, you can but it's recommended to [run some samples](https://aneoconsulting.github.io/ArmoniK/installation/linux/verify-installation#samples) in ArmoniK to make sure that the service will return some data.
+::
+
+### Create the service
+
+First, we need to create a folder to store our services.
+
+```bash
+mkdir src/app/services
+```
+
+Then, we can create a new file `src/app/services/partitions-grpc.service.ts` and create a service:
+
+```typescript [partitions-grpc.service.ts]
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class PartitionsGrpcService {
+}
+```
+
+::alert{type="info"}
+In Angular, it's a convention to add the `.service` suffix in the filename and to name the service with the suffix `Service`.
```

Comment:
In Angular, it is a convention to add the `.service` suffix in the filename and to name the service with the suffix `Service`.